en/identity-server/next/docs/apis/restapis/notification-senders-v2.yaml

```diff
@@ -0,0 +1,1354 @@
+openapi: 3.0.0
+info:
+  title: WSO2 Identity Server - Notification Senders API Definition
+  description: This document specifies a **RESTful API** for **WSO2 Identity Server Notification Senders**
+  contact:
+    name: WSO2
+    url: http://wso2.com/products/identity-server/
+    email: architecture@wso2.org
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+servers:
+  # Added by API Auto Mocking Plugin
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/abc-70e/notification-sender/1.0.0
+  - url: https://localhost:9443/t/{tenant-domain}/api/server/v1
```

Comment:
Do we access API in both root orgs and sub orgs with the same URL? If not we may need to add a separate yml file for suborg APIs.